aduro/helpers.py

```diff
@@ -0,0 +1,23 @@
+"""Helper methods for the Seluxit API."""
+from typing import Any
+
+
+def try_convert_object(obj: Any, field_name: str) -> Any:
+    """
+        Try convert the object to the correct value.
+        It is unused but I keep this when I find out how to get `pydantic` to use custom parser.
+    """
+    orig_value = obj.get(field_name)
+    if orig_value is None:
+        return None
+    if "." in orig_value:
+        value = float(orig_value)
+    elif orig_value.lower() == "true":
+        value = True
+    elif orig_value.lower() == "false":
+        value = False
+    elif orig_value.isdigit():
+        value = int(orig_value)
+    else:
+        value = orig_value
+    return value
```

Comment:
Should this return a value that we don't know what is, or instead throw an exception?

Reply:
I would like it to return value anyhow, but maybe convert this to str in this case?

Reply:
That seems safer, and then you can also use Union[int, float, bool, str, ...] for the return type.